src/patternfly/components/Select/examples/select-single-top-expanded-example.hbs

```diff
@@ -0,0 +1,4 @@
+<div style="padding:130px"></div>
```

Comment:
Users should be able to copy and paste the example code from the workspace into their application, so we try to avoid unnecessary elements with inline styles like this.

Have a look at how the top aligned dropdown example is setup.

https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Dropdown/examples/index.js#L70-L72

https://github.com/patternfly/patternfly-next/blob/master/src/site/workspace.scss#L424-L428

Reply:
Thanks!